analyzers/its/expected/akka.net/S1144-Akka.MultiNodeTestRunner.Shared-netstandard2.0.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Issues": [
+    {
+      "Id": "S1144",
+      "Message": "Remove the unused private setter \u0027set_TestRunCompleted\u0027.",
+      "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/akka.net/src/core/Akka.MultiNodeTestRunner.Shared/Reporting/TestRunCoordinator.cs#L80",
```

Comment:
This might be a FP

Reply:
The comment says that the property is automatically set when "EndTestRun" is sent to this actor. However, it is still not clear to me what this means exactly

Reply:
I don't see how this would be possible.